components/forms/DelegationForm.tsx

```diff
@@ -93,7 +93,7 @@ const DelegationForm = (props: Props) => {
           value={validatorAddress}
           onChange={(e: React.ChangeEvent<HTMLInputElement>) => setValidatorAddress(e.target.value)}
           error={addressError}
-          placeholder={`E.g. cosmosvaloper1sjllsnramtg3ewxqwwrwjxfgc4n4ef9u2lcnj0`}
+          placeholder={`E.g. ${state.chain.addressPrefix}1sjllsnramtg3ewxqwwrwjxfgc4n4ef9u2lcnj0`}
```

Comment:
Let's get some `exampleValidatorAddress` function just like `exampleAddress` and use it here. We need to explain the user to use the `XXXXvaloper1YYYYYY` address somehow.

Reply:
There are chains that seem to use `valoper` for validator addresses but don't set it on the chain json, so we can't know beforehand

Reply:
Almost all chains use valoper addresses. Tgrade is an exception here.

So we can just guess it will be `XXXXvaloper1YYYYYY` like this:
- Take some ranodom 20 bytes
- Use `const validatorPrefix = prefix + valoper`
- call `toBech32` to generate a full example address with the two inputs

Reply:
The validation can remain less strict but the example should hint to valoper addresses